.github/workflows/documentation.yml

```diff
@@ -0,0 +1,54 @@
+# This is a workflow to generate the documentation automatically.
+
+name: Generate Documentation
+
+# Controls when the action will run. Triggers the workflow on push
+# events but only for the master branch
+on:
+  push:
+    branches:
+      - master
+      - feature/AutomaticDocDeployment # this will be removed once it's working
+jobs:
+  # This workflow contains a single job called "build"
+  gen_doc:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
```

Comment:
why not `ubuntu-latest`?

Reply:
Actually I don't know. It works with latest, but the [doc](https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-keeping-existing-files) of the tool says 18.04